packages/components/src/components/Badge/Badge.tsx

```diff
@@ -1,73 +1,43 @@
 import React from 'react';
-import styled, { css, DefaultTheme } from 'styled-components';
+import styled, { css } from 'styled-components';
 import { IconName } from '@suite-common/icons';
 import { Icon } from '@suite-common/icons/src/webComponents';
 import { borders, Color, spacings, spacingsPx, typography } from '@trezor/theme';
 import { focusStyleTransition, getFocusShadowStyle } from '../../utils/utils';
+import type { UISize, UIVariant } from '../../config/types';
 
-const getBackgroundColor = (variant: BadgeVariant | undefined, theme: DefaultTheme) => {
-    switch (variant) {
-        case 'green':
-            return theme.backgroundPrimarySubtleOnElevation0;
-        case 'red':
-            return theme.backgroundAlertRedSubtleOnElevation0;
-        case 'bold':
-            return theme.backgroundNeutralBold;
-        case 'neutral':
-        default:
-            return theme.backgroundNeutralSubtleOnElevation0;
-    }
-};
+type BadgeSize = Extract<UISize, 'tiny' | 'small' | 'medium'>;
```

Comment:
I like the idea of extracting the "supported" values.  But I would strictly forbid adding a new one.

Local `BadgeSize` shall be always subset of `UISize`

Reply:
but how to forbid it?

Reply:
Probably we have to do it just by convention and code-review.